scripts/copy_cromwell_import_to_bucket_puddle.py

```diff
@@ -21,6 +21,9 @@ def phef(f):
     ("pheweb_import.annotation.gene_trie",lambda x: f'/cache/'),("pheweb_import.annotation.sites",phef),
     ("pheweb_import.annotation.trie1",phef),("pheweb_import.annotation.trie2",phef)]
 
+
+output_nodes =[ ("pheweb_import.matrix.matrix",phef)]
```

Comment:
why do you overwrite with not outputting anything else but this?

Reply:
not even sure i touched this script ever..

Reply:
this change is in your commit it seems. maybe you tested it and it was forgotten there... anyhoo remove this line